dotCMS/src/main/java/com/dotcms/rest/api/v1/system/UpgradeTaskForm.java

```diff
@@ -0,0 +1,39 @@
+package com.dotcms.rest.api.v1.system;
+
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.dotcms.repackage.org.hibernate.validator.constraints.Length;
+import com.dotcms.rest.api.Validated;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(builder = UpgradeTaskForm.Builder.class)
+public class UpgradeTaskForm extends Validated {
+
+    @NotNull
+    @Length(min = 2, max = 255)
+    private final String upgradeTaskClass;
+
+    public String getUpgradeTaskClass() {
+        return upgradeTaskClass;
+    }
+
+    private UpgradeTaskForm(final Builder builder) {
+        this.upgradeTaskClass = builder.upgradeTaskClass;
+        checkValid();
+    }
+
+    public static final class Builder {
+
+        @JsonProperty(required = true) private String upgradeTaskClass;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field upgradeTaskClass has the same name as a method](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4324168)